texastribune/queso-ui
# Standardize blast header component

#### What's this PR do?
Copies [The Blast header](https://www.texastribune.org/theblast/) styles into our shared lib.

##### Classes added (if any)
- c-blast-header

#### Why are we doing this? How does it help us?
We use this Blast-branded header in both our /texastribune and /donations repos. This feels like a good candidate for a shared component.

#### How should this be manually tested?
`yarn dev`

See: [c-blast-header standalone view](http://localhost:3000/pages/components/c-blast-header/raw-preview.html)


#### Does this introduce a breaking change where queso-ui is used in the wild? If so, is there a relevant branch/PR to accompany this release?

No breaking changes, but there is a tt PR to accompany it:
https://github.com/texastribune/texastribune/pull/3613


#### TODOs / next steps:

* [ ] *Convert [The Blast landing pages](https://www.texastribune.org/theblast/) in the tt repo to use this*
